source/java/product-guide/index.markdown

```diff
@@ -2899,7 +2899,180 @@ A request returns a paginated list of the group memberships where the group is i
     }
 
 ***
+<a class="anchor" name="organizations"></a>
 
```

Comment:
@dogeared This is the updated organization resource documentation in case you want to take a look. Thanks!